unit_tests/test_bit_field_original_value.py

```diff
@@ -7,40 +7,40 @@
 
 @given("A BitField")
 def request_one_block(context):
-    context.uut = BitField(100, width=8)
+    context.uut = BitField(default_value=100, width=8, name="one_block")
 
 
 @given("A 4 byte BitField with value 100")
 def bitfield_ascii_100(context):
-    context.uut = BitField(100, width=32)
+    context.uut = BitField(default_value=100, width=32, name="ascii_100")
 
 
 @given(parsers.parse("A 4 byte BitField with value {value:d} and format ascii"))
 def bitfield_4_bytes(context, value):
-    context.uut = BitField(value, width=32, output_format="ascii")
+    context.uut = BitField(default_value=value, width=32, output_format="ascii", name="4_bytes")
 
 
 @given("Mutated once")
 def mutate_once(context):
-    context.uut.mutate()
+    next(context.uut.get_mutations())
 
 
 @given("Mutated twice")
 def mutate_twice(context):
-    context.uut.mutate()
-    context.uut.mutate()
+    next(context.uut.get_mutations())
+    next(context.uut.get_mutations())
 
 
 @given("Mutated thrice")
 def mutate_thrice(context):
-    context.uut.mutate()
-    context.uut.mutate()
-    context.uut.mutate()
+    next(context.uut.get_mutations())
+    next(context.uut.get_mutations())
+    next(context.uut.get_mutations())
 
 
 @when("Calling original_value")
 def call_original_value(context):
-    context.result = context.uut.original_value
+    context.result = context.uut.original_value().to_bytes(1, "little")
```

Comment:
Doesn't this do the same as `render()` or `encode()`?
Seems related to https://github.com/jtpereyda/boofuzz/pull/422#issuecomment-666663935. This doesn't feel like a very robust rest.
It might work in this particular case but maybe we should drop the original_value tests completely after all @jtpereyda?
As I mentioned in the comment above, I don't think these tests are very meaningful. All we could do is test a few selected cases where we expect a given outcome.

Reply:
Testing basic functionality to see if it's doing what you expect sounds valuable to me.
Maybe it's what `render()` or `encode()` does, but the test will let us know if it's accidently broken.

Reply:
Alright I'm convinced.
It does work fine for this and the string primitive. Definitely better than nothing.